### PR TITLE
DOC: Reduced main image size

### PR DESCRIPTION
**Pull request by viv511**
Scaled original image by 50% in height and width. Changed because original image was too large.

Closes PR #4785 

## PR Checklist
- [x] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [x] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [x] Added test (or behavior not changed)
- [x] Updated API documentation (or API not changed)
- [x] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [x] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [x] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples) for all new major features (if any)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

**Review by jhlegarreta — COMMENTED** [review]
IMO the image should have a fixed size: what if a new version of the logo that has a different size is added?

**Comment by viv511** [comment]
Hi! Sure, I can add a new version of the logo with a different size. I used a HTML/CSS fix as [per discussion](https://github.com/InsightSoftwareConsortium/ITK/issues/4785#issuecomment-2248765436) in the original issue thread

**Comment by jhlegarreta** [comment]
> Hi! Sure, I can add a new version of the logo with a different size. I used a HTML/CSS fix as https://github.com/InsightSoftwareConsortium/ITK/issues/4785#issuecomment-2248765436 in the original issue thread

I did not mean that we would need to add a logo with a different size; the issue probably arised because the new logo is much larger than the previous one, and if the situation repeats in the future the `50%` width/height solution may be problematic. So IMO ideally we want the size of the displayed logo be always the same size (or have a fixed width or height, to respect its aspect ratio), regardless of the size of the original file.

**Review by dzenanz — APPROVED** [review]

**Comment by dzenanz** [comment]
ITK's logo does not change frequently. This PR improves the situation. Further improvements can be done in a follow-up PR, and/or when the logo is changed again.

File: Documentation/Doxygen/MainPage.dox [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *
  * \mainpage  Insight Toolkit
  *
- * \image html itkLogo.png
+ * \image html itkLogo.png width=50% height=50%
```
Comment (dzenanz) [inline Documentation/Doxygen/MainPage.dox:5]: Is this 50% of the image's original size, or 50% of screen/page size?

Comment (viv511) [reply Documentation/Doxygen/MainPage.dox:5]: This is 50% of the image's original size. The image size is relative to the original size of the image itself, not the screen or page size.

Thank you for reviewing!

**Pull request merged**